pkg/kuttlctl/cmd/values.go

```diff
@@ -0,0 +1,30 @@
+package cmd
```

Comment:
I called it values because the type implements a`pflag.Value` [interface](https://pkg.go.dev/github.com/spf13/pflag@v1.0.5#Value) and I thought it might be a good name if we add more types that can be used via that interface. Maybe I should have inserted a comment about this...

But sure, this can be `labels.go` too if we go for one file per type. Let me know which one you prefer.